README.md

```diff
@@ -32,3 +32,18 @@ docker run -d --name asyncapi-playground -p 83:5000 asyncapi-playground:latest
 
 Then browse to [http://localhost:83/]()
 
+## Load async api
+
+2 ways to load async apis from an url: 
+
+- Use the *load* parameter.
+
+```
+https://playground.asyncapi.io/?load=https://raw.githubusercontent.com/asyncapi/asyncapi/master/examples/2.0.0/simple.yml
+```
+
+- Use the *url* parameter.
+```
+https://playground.asyncapi.io/?url=https://raw.githubusercontent.com/asyncapi/asyncapi/master/examples/2.0.0/simple.yml
+```
```

Comment:
How about something like 👇 

## Load AsyncAPI with URL parameter

There are two parameters that serve the same purpose:

- **load** parameter:
   ```
   https://playground.asyncapi.io/?load=https://raw.githubusercontent.com/asyncapi/asyncapi/master/examples/2.0.0/simple.yml
   ```
- **url** parameter:
   ```
   https://playground.asyncapi.io/?url=https://raw.githubusercontent.com/asyncapi/asyncapi/master/examples/2.0.0/simple.yml
   ```

Reply:
Totally ok, no problems :)